deprecated/classic.py

```diff
@@ -29,7 +28,7 @@
     else:
         _class_stacklevel = 3
 
-string_types = (type(b''), type(u''))
+string_types = (bytes, str)
```

Comment:
With Python 2.7, it was common practice to use a definition such as `string_types = (bytes, str)` to manage `str` and `byte` arrays. However, this backward compatibility is no longer necessary, as Python 3 has clearly differentiated between `str` (Unicode text) and `bytes` (binary data).
To modernize the code, we should eliminate `string_types` and use `str` directly.